### PR TITLE
enable backups on all Droplets

### PR DESCRIPTION
**Pull request by afeld**
Builds on #18.

### CODE REVIEW

File: tasks/droplet.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      uri:
+        url: https://api.digitalocean.com/v2/droplets/{{ do_id }}/actions
+        method: POST
+        headers:
+          Authorization: Bearer {{ lookup('env', 'DO_API_TOKEN') }}
+        body_format: json
+        body:
+          type: enable_backups
+        status_code: 201
+      when: "'backups' not in droplet_info.json.droplet.features"
```
Comment (allthesignals) [inline tasks/droplet.yml]: Interesting - is this just a console message or?

**Review by allthesignals — APPROVED** [review]
Is the DO token required only for this step? Or does that not matter as much since this is nightly.

File: tasks/droplet.yml (hunk 1 above)
Comment (afeld) [reply tasks/droplet.yml]: No, it's a [conditional](https://docs.ansible.com/ansible/devel/user_guide/playbooks_conditionals.html#the-when-statement). It only enables backups if not previously enabled.

**Comment by afeld** [comment]
Rebased and added some comments. Do the latter help clarify?

**Pull request merged**